boltdesignsystem/bolt
# Migrate Carousel to render using LitElement (BoltElement)

## Jira
http://vjira2:8080/browse/BDS-2109

## Summary
Migrates Carousel over to use LitElement for rendering without any major UI logic changes otherwise. 

## Details
Not only does this address the major rendering issues I ran into while implementing the new `lazy-queue` functionality globally, this should also make pages with many Carousel instances render much faster!

## How to test
- Review code changes
- Spot check existing Carousel instances to see if there's any unforeseen side-effects from these updates
